main.js

```diff
@@ -0,0 +1,11 @@
+const { app, BrowserWindow } = require("electron");
+
+let mainWindow = null;
+
+app.on("ready", () => {
+  mainWindow = new BrowserWindow();
+  mainWindow.loadFile(__dirname + "/src/html/login.html");
```

Comment:
Please put comment or add information in README that the path must be modified for Personium Cell or Electron.